extensions/cornerstone/src/commandsModule.ts

```diff
@@ -68,16 +92,151 @@ const commandsModule = ({ servicesManager }) => {
       toolGroupIdToUse = toolGroup.id;
     }
 
-    const toolGroup = ToolGroupService.getToolGroup(toolGroupIdToUse);
+    const toolGroup = toolGroupService.getToolGroup(toolGroupIdToUse);
     return toolGroup;
   }
 
+  function _showViewerContextMenu(viewerElement, options) {
+    let defaultPointsPosition = [];
+    if (options.nearbyToolData) {
+      defaultPointsPosition = commandsManager.runCommand(
+        'getToolDataActiveCanvasPoints',
+        { toolData: options.nearbyToolData }
+      );
+    }
+
+    contextMenuController.showContextMenu(
+      options,
+      viewerElement,
+      defaultPointsPosition
+    );
+  }
+
   const actions = {
+    /** Show the specified context menu */
+    showViewerContextMenu: providedOptions => {
```

Comment:
isn't this name redundant? do we have a non-viewer context menu?

Reply:
Not yet, but I would guess that we will have them - for example, a context menu on the measurements in the measurements table.